build/gh-pages-app.js

```diff
@@ -3,20 +3,21 @@
  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
-'use strict';
-
-const fs = require('fs');
-const path = require('path');
-const rollup = require('rollup');
-const rollupPlugins = require('./rollup-plugins.js');
-const cpy = require('cpy');
-const ghPages = require('gh-pages');
-const glob = require('glob');
-const lighthousePackage = require('../package.json');
-const terser = require('terser');
-const {LH_ROOT} = require('../root.js');
+
+import fs from 'fs';
+import path from 'path';
+
+import {rollup} from 'rollup';
+import cpy from 'cpy';
+import ghPages from 'gh-pages';
+import glob from 'glob';
+import * as terser from 'terser';
+
+import * as rollupPlugins from './rollup-plugins.js';
+import {LH_ROOT, readJson} from '../root.js';
 
 const ghPagesDistDir = `${LH_ROOT}/dist/gh-pages`;
+const lighthousePackage = readJson(`${LH_ROOT}/package.json`);
```

Comment:
Does this affect how we read package.json in the bundle: 

https://github.com/GoogleChrome/lighthouse/blob/0fcf4a78aed37bad1aa71691c74154ec23c91444/build/build-bundle.js#L119-L120

Reply:
It does not, but it should be replaced now.